test.cpp

```diff
@@ -348,16 +348,18 @@ class Test8 {
               tmp *= rand();
       };
 
+	  pid_t tid = syscall(SYS_gettid);
+
```

Comment:
in test8, Collection is created without enablePerapiPerf enabled. (see Line 322)
collect_scope_start/stop for per API performance does work?

Reply:
I have refactored the test case and have made it worked. But I removed all of background threads since we really don't want to capture any background threads cycles for per api. It's only designed for profiling main thread performance.